apps/ejabberd/test/ejabberd_sm_SUITE.erl

```diff
@@ -203,8 +205,31 @@ kv_can_be_updated_for_session(C) ->
     when_session_info_stored(U, S, R, {key2, override}),
 
     [#session{sid = Sid, info = [{key1, val1}, {key2, override}]}]
-     = ?B(C):get_sessions(U,S).
-
+     = ?B(C):get_sessions(U, S).
+
+cannot_reproduce_race_condition_in_store_info(C) ->
```

Comment:
Finally I modified the test case to work with both backends. Now it's run in both groups and there is no need to create new one.